src/dispatcher/server.rs

```diff
@@ -135,7 +146,10 @@ impl SignatureDispatcher for Ed25519SignatureDispatcher {
         }
 
         let reply = SignMilestoneResponse {
-            signatures: signatures.iter().map(|signature| signature.as_ref().unwrap().get_ref().to_owned().signature).collect()
+            signatures: signatures
+                .iter()
+                .map(|signature| signature.as_ref().unwrap().get_ref().to_owned().signature)
+                .collect(),
         };
 
         info!("Successfully signed.");
```

Comment:
Let's only log this if we actually have at least one signature produced! Now you can actually get something like this:
```
2021-02-08 15:00:48,602 DEBUG [dispatcher_server] Got Request: Request { metadata: MetadataMap { headers: {"te": "trailers", "content-type": "application/grpc"} }, message: SignMilestoneRequest { pub_keys: [[213, 120, 208, 168, 229, 57, 32, 64, 207, 92, 59, 160, 21, 60, 40, 195, 0, 216, 234, 237, 61, 141, 126, 244, 55, 41, 202, 223, 225, 226, 70, 123], [38, 135, 131, 231, 39, 124, 124, 159, 109, 200, 152, 208, 143, 92, 164, 88, 148, 29, 226, 235, 51, 158, 125, 169, 72, 35, 156, 195, 100, 125, 255, 204]], ms_essence: [83, 105, 103, 110, 32, 116, 104, 105, 115, 33] }, extensions: Extensions }
2021-02-08 15:00:48,603 INFO  [dispatcher_server] Got Request that matches signers: Map { iter: Map { iter: Unique { iter: UniqueBy { iter: Iter([]), used: {[38, 135, 131, 231, 39, 124, 124, 159, 109, 200, 152, 208, 143, 92, 164, 88, 148, 29, 226, 235, 51, 158, 125, 169, 72, 35, 156, 195, 100, 125, 255, 204]: (), [213, 120, 208, 168, 229, 57, 32, 64, 207, 92, 59, 160, 21, 60, 40, 195, 0, 216, 234, 237, 61, 141, 126, 244, 55, 41, 202, 223, 225, 226, 70, 123]: ()} } } } }
2021-02-08 15:00:48,603 INFO  [dispatcher_server] Successfully signed.
2021-02-08 15:00:48,603 INFO  [dispatcher_server] MS Essence: [83, 105, 103, 110, 32, 116, 104, 105, 115, 33]
2021-02-08 15:00:48,603 INFO  [dispatcher_server] Used Signers: []
2021-02-08 15:00:48,603 INFO  [dispatcher_server] Signatures: []

```

Reply:
`connect_signer` method should have thrown an error at this point.

Reply:
We should refactor this long function in a seperate PR